main/ui/src/main/resources/fxml/forget_password.fxml

```diff
@@ -8,9 +9,8 @@
 <?import javafx.scene.layout.StackPane?>
 <?import javafx.scene.layout.VBox?>
 <?import javafx.scene.shape.Circle?>
-<?import org.cryptomator.ui.controls.FontAwesome5IconView?>
-<VBox xmlns="http://javafx.com/javafx"
-	  xmlns:fx="http://javafx.com/fxml"
+<VBox xmlns:fx="http://javafx.com/fxml"
+	  xmlns="http://javafx.com/javafx"
```

Comment:
can we have default namespace before explicit namespaces?

Reply:
As far as I can see after googling and testing, there is no way to sort the namespaces like that. I can only disable sorting/remove the rule `xmlns`, but that would only lead to inconsistencies in my eyes.

Reply:
fair enough, we'll keep this.